.github/workflows/release.yml

```
@@ -0,0 +1,77 @@
+name: release
+
+on:
+    workflow_dispatch:
+        inputs:
+            RELEASE_VERSION:
+                description: 'Version to release'
+                required: true
+
+jobs:
+    release:
+        name: Release ${{ github.event.inputs.RELEASE_VERSION }}
+        runs-on: ubuntu-latest
+
+        steps:
+            - name: Checkout repo
+              uses: actions/checkout@v4
+            
+            - name: Install uv
+              uses: astral-sh/setup-uv@v5
+
+            - name: Install dependencies
+              run: |
+                uv venv --python=3.11
+                uv pip install -r requirements/ci.txt -r pipeline/requirements.txt
+
+            - name: Download raw data
+              run: |
+                uv run make download-raw
+
+            - name: Process raw data into duckdb file
+              run: |
+                uv run make process
+
+            - name: Set environment variable for version
+              run: |
+                echo "VERSION_UNDERSCORE=${{ github.event.inputs.RELEASE_VERSION }}" | sed 's/\./_/g' >> $GITHUB_ENV
```

```suggestion
                echo "VERSION_UNDERSCORE=${{ github.event.inputs.RELEASE_VERSION }}" | sed 's/\./_/g' | tee -a $GITHUB_ENV
```

Using `tee` here is slightly nicer for debugging because it also prints out to console.